core/data_masking/abstract_mask.py

```diff
@@ -0,0 +1,52 @@
+from abc import ABC
+
+
+class DataMaskAbs(ABC):
+    anon_fields = []
+    masking_model = None
+    enabled = False
+
+    def apply_mask(self, data):
+        for field in self.anon_fields:
+            field_parts = field.split('.')
+            self._anonymize_nested_field(data, field_parts)
+        return data
+
+    def anonymize(self, value):
+        if isinstance(value, str):
+            result = '***************'
+            return result
+        elif isinstance(value, (int, float)):
+            return 0
```

Comment:
Are you sure you want to anonymize numbers using 0? It might be taken as a correct value by mistake.

Reply:
I'll fix this in next PR, I also have to add masking for date and datetime values too